src/components/molecules/NavMenu/styles.tsx

```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components"
+import { space } from "../../../styles/vars"
+
+export const StyleMenuLinkContainer = styled.div`
+    height: 50vh;
+    margin: ${space.l};
+    min-height: 375px;
+
+    @media (min-width: 768px) {
```

Comment:
why are you doing this?

This should be something like:

```
 ${media.med`
  
  `}
```
(Don't forget to import media)

Reply:
Weird. My version doesn't look like that. It has media.med like you wrote.